src/ProjectTemplates/ComponentsWebAssembly.ProjectTemplates/content/ComponentsWebAssembly-CSharp/Client/Program.cs

```diff
@@ -8,6 +8,7 @@
 #endif
 using Microsoft.AspNetCore.Components.WebAssembly.Hosting;
 using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
```

Comment:
I think this should be sorted after the DI namespace.

Reply:
I swear I know how to spell.

![image](https://media3.giphy.com/media/G4Ihli2UThrBS/giphy.gif?cid=ecf05e47e201863a441c1a80c96eec54590163d168b2eb9f&rid=giphy.gif)